fgrpc/pingsrv.go

```diff
@@ -92,11 +94,14 @@ func PingServer(port, cert, key, healthServiceName string, maxConcurrentStreams
 
 // PingClientCall calls the ping service (presumably running as PingServer on
 // the destination). returns the average round trip in seconds.
-func PingClientCall(serverAddr, cacert string, n int, payload string, delay time.Duration) (float64, error) {
+func PingClientCall(serverAddr, cacert string, n int, payload string, payloadSize int, delay time.Duration) (float64, error) {
```

Comment:
no, this is weird to pass 2 payloads to this function. don't change this signature (will also avoid changing a bunch of tests), instead generate from the caller